packages/fluentui/docs/src/examples/components/Card/Usage/CardExampleSelectableGrid.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  Flex,
+  Image,
+  Text,
+  Card,
+  cardSelectableBehavior,
+  Grid,
+  cardsContainerBehavior,
+  Checkbox,
+  Button,
+  screenReaderContainerStyles,
+  hiddenComponentBehavior,
+} from '@fluentui/react-northstar';
+import * as React from 'react';
+import * as _ from 'lodash';
+
+type SelectableCardProps = {
+  index?: number;
+  title?: string;
+  selected?: boolean;
+  onClick?: Function;
+};
+
+const SelectableCard: React.FC<SelectableCardProps> = ({ title, index, selected, onClick, ...unhadledProps }) => {
```

Comment:
```suggestion
const SelectableCard: React.FC<SelectableCardProps> = ({ title, index, selected, onClick }) => {
```

I would avoid having there `unhadledProps`, let's have:
- all properties specified
- call it `rest`

Reply:
Reason: `unhadledProps` are coming from `useUnhandledProps` and are used in our components. However, in this case it's just rest/spread